packages/subgraph/abis/GardensTemplate.json

```diff
@@ -0,0 +1,260 @@
+[
```

Comment:
With the changes on #64 we can remove this one as we are using the local version instead.